File: CSharp part2/03. Methods/03.EnglishDigit/EnglishDigit.cs
// Write a method that returns the last digit of given integer as an English word.

using System;

class EnglishDigit
{
    static string LastDigitAsWord(int number)
    { 
        int lastDigit = number % 10;
        string result = "";
        switch (lastDigit)
        {
            case 0: result = "zero" ; break;
            case 1: result = "one"; break;
            case 2: result = "two"; break;
            case 3: result = "three"; break;
            case 4: result = "four"; break;
            case 5: result = "five"; break;
            case 6: result = "six"; break;
            case 7: result = "seven"; break;
            case 8: result = "eight"; break;
            case 9: result = "nine"; break;
        }
        return result;
    }
    static void Main()
    {
        Console.Write("Enter a number:");
        int number = int.Parse(Console.ReadLine());
        Console.WriteLine("Last digit as word:{0}",LastDigitAsWord(number));
    }
}

File: JavaScript-Fundamentals/Functions/01.EnglishDigit/EnglishDigit.js
// Write a function that returns the last digit of given integer as an English word.

var jsConsole;
var btn = document.getElementById('returnDigit');
btn.onclick = returnDigit;

function returnDigit () {
	var input = jsConsole.readInteger('#n');
	var lastDigit = input % 10;
	switch (lastDigit) {
		case 0:
			jsConsole.writeLine("zero");
			console.log("zero");
			break;
		case 1:
			jsConsole.writeLine("one");
			console.log("one");
			break;
		case 2:
			jsConsole.writeLine("two");
			console.log("two");
			break;
		case 3:
			jsConsole.writeLine("three");
			console.log("three");
			break;
		case 4:
			jsConsole.writeLine("four");
			console.log("four");
			break;
		case 5:
			jsConsole.writeLine("five");
			console.log("five");
			break;
		case 6:
			jsConsole.writeLine("six");
			console.log("six");
			break;
		case 7:
			jsConsole.writeLine("seven");
			console.log("seven");
			break;
		case 8:
			jsConsole.writeLine("eight");
			console.log("eight");
			break;
		case 9:
			jsConsole.writeLine("nine");
			console.log("nine");
			break;
		default:
			jsConsole.writeLine("not a digit");
			console.log("not a digit");
	}
}

